### ISSUE TITLE
Support for query by multiple hex_signatures

### ISSUE THREAD

**Issue by devtooligan** [pipermerriam/ethereum-function-signature-registry#101]
Hi!
This is a really great tool. thanks so much for doing this. 

Would you be open to me submitting a pull request to allow for querying by multiple hex_signatures in one request?

something like:
```
?hex_signatures__in=0xabc123,0xdef456
```

Thanks again!

**Comment by pipermerriam** [comment]
Yes, I could get onboard with that.  PR should include tests and updates to the API docs pages (which are also hosted by the application)

**Comment by devtooligan** [comment]
Awesome! [I've submitted a pr here](https://github.com/pipermerriam/ethereum-function-signature-registry/pull/102)
